src/main/java/space/obminyashka/items_exchange/dto/UserDto.java

```diff
@@ -27,6 +27,7 @@ public class UserDto {
     @Email(message = ResponseMessagesHandler.ValidationMessage.INVALID_EMAIL)
     private String email;
     private boolean online;
+    private boolean oauth2login;
```

Comment:
All the boolean variables have to contain some prefix either **is** or **has** that would allow us to better understand the type and expected result.

Reply:
@Sershok check my comment one more time

Reply:
@Sershok check the comment one more time.